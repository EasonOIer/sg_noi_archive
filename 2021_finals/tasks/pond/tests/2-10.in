1941 968
998496 997492 995045 994778 993519 993514 993033 992498 992384 990065 988895 988012 981081 980692 980665 978701 976899 976692 975304 973759 972128 971463 966298 965444 962935 962653 962060 961689 959906 959584 958258 958143 957777 957506 955245 955180 954622 954285 954265 952947 950835 950018 948402 947322 947260 946786 943660 942642 938699 937894 937457 936733 936326 935934 933922 933780 932710 931842 930437 929397 928981 927698 926655 926444 925503 924738 924196 924023 923604 921202 919977 918633 917985 917764 917733 917319 916641 916517 916340 915618 913584 913363 912485 911842 911695 911549 909199 908639 904048 903829 903785 903353 902772 901304 900859 898165 897206 894856 894853 893964 892417 892022 891884 891813 891429 890728 890556 888637 888316 886724 885482 884666 883975 881751 880610 880265 880120 878363 878096 876630 875162 874625 870904 869231 868050 865192 864870 864317 864040 863921 863343 863298 863071 862354 862219 860389 859687 859061 858024 857710 854916 854667 851573 851469 848150 847721 847510 846560 844250 843584 843103 842461 840975 839116 837798 837591 834113 832510 831681 831130 829422 828383 828161 827079 823525 822869 819407 819184 818179 816665 816264 816234 815240 813628 813497 813234 813046 812441 811981 811863 810720 809323 808282 803468 801450 801268 799757 799129 796414 792279 791104 791076 790766 790421 788725 788291 785403 783857 783646 782662 782264 781053 780171 778728 777419 775855 775750 774992 772132 772113 770899 770259 767041 765176 764881 762659 761933 761892 761525 760786 759469 758425 758217 757591 753878 753803 753288 752765 750562 750151 749178 747137 746833 746262 745704 745535 745534 745286 744870 744538 741430 741189 738001 735575 735555 735285 734155 733243 732382 731611 731452 731008 729358 725285 725107 722959 719630 716530 716402 712056 711267 711060 710815 709994 709561 707647 707487 706519 706041 703873 703027 702963 701486 698377 698376 696887 695276 694893 694228 693979 693928 691111 690941 689584 688586 686932 686483 686322 685177 685159 684285 681598 680875 677584 677495 676190 676083 675943 673986 673507 672207 672093 672070 671741 671514 671287 670532 667997 667849 666829 666501 663175 659791 657543 657112 655993 654778 654218 652810 651705 651158 650788 649485 648213 648005 647422 646154 643485 642060 635530 635436 634781 634138 633213 633034 632975 632781 632209 632154 632094 631718 629868 629508 628997 627970 627461 627322 626939 626295 625356 625343 625189 624520 624408 622050 620894 620696 620242 619172 617226 616883 616482 616186 615887 615689 614174 614019 612767 611885 611690 611634 608515 608229 607957 607907 606273 605177 605047 601862 601564 600978 599820 599491 598378 598315 598151 597812 597030 592590 592283 590011 589659 589543 589372 588999 586960 585593 584628 584256 583147 583136 581251 578786 577781 577645 576406 576052 574066 573947 573039 569389 568996 568644 566080 565581 563712 563671 563143 562090 554495 554153 553876 553366 551716 551520 551261 550962 550649 550320 548915 548622 546223 546127 545820 545167 544967 543053 541906 541491 541476 540877 540376 538611 538201 537542 536117 535964 532541 531879 531477 530269 529328 528905 527374 526969 524519 524272 523106 523023 523007 522486 522072 521450 521400 520777 520756 517868 517709 517369 513971 512740 511331 509854 509181 505135 504632 504397 503852 500055 499072 498790 498706 498348 497751 496810 496790 496575 493709 492699 491904 491697 491037 490751 488727 486749 486497 486192 486020 485962 484508 479816 479028 477877 476362 475659 474697 474581 474568 473445 472872 472009 469964 469939 468774 468616 466335 466262 466174 464822 458730 457008 456786 456258 456025 454848 451604 451125 450914 450591 450567 449653 449422 448939 447837 447650 447448 446409 444009 443785 443071 441479 440874 439137 439009 438920 438313 438188 437421 435467 435392 435176 433703 433548 432666 431457 431439 431399 431029 427660 427459 427254 426425 426096 424660 421760 421612 417780 417029 416728 415551 410484 409957 409604 409181 409087 408766 408640 407130 406104 406065 405821 405702 405455 405131 404567 404266 404240 401513 401102 400379 398539 397710 397286 396595 396466 396171 395723 394694 394343 393106 392558 391627 389902 388806 388554 388514 386503 386262 385703 385002 384378 381736 381447 380258 379570 378770 378749 378578 376537 375543 374649 374282 370565 370086 369897 368810 368373 366941 366561 366239 366166 365119 364526 361595 361559 357269 357243 355931 355166 354329 352582 352306 352278 350265 350094 349499 346412 346408 345465 344629 344199 342920 341790 341453 340221 339814 339801 337929 335074 334335 333939 333644 332447 329691 329184 328697 328696 328264 325160 324854 322268 322232 321469 321142 318426 316831 315125 315055 313035 312999 312063 311977 311687 311269 310882 310707 307627 306628 303517 302889 302708 301847 301441 300797 300663 299912 296876 296013 295953 294711 294449 294236 292568 291644 290362 289781 288535 287892 287217 284127 283683 281951 281063 280406 280170 279901 278724 278721 276005 275651 272908 271220 270702 269356 269119 268696 266560 266281 265366 264145 263387 262473 260545 260312 259958 258945 258132 256623 256479 254671 253915 253525 252668 252588 252137 251529 248515 248220 248010 246142 244193 244093 243950 242712 239471 237128 236887 236478 236286 235555 234774 234215 234184 232147 228156 225752 222893 222826 222094 221799 220775 218340 218094 217672 216714 215727 214804 214711 214406 213413 213302 213183 213161 212915 208716 208108 207645 206941 206145 204793 204361 202807 201809 201655 201614 201291 201047 199872 199787 197482 197207 196831 196287 191958 189765 187107 186952 185589 184803 183552 182866 182346 182167 181697 181465 180353 179793 175942 175459 174383 174116 172514 172036 171079 169535 167640 167522 164890 164211 159840 159402 159099 151268 148242 148026 148008 144707 143202 142566 141679 141505 141320 141236 139474 138171 137268 137044 134909 134723 134656 131846 131774 127812 126758 125645 122561 121840 121217 120429 117633 116825 112728 112595 110840 110229 108723 107377 105932 105469 104711 103417 102347 100863 98002 93450 92897 92284 92185 90244 89909 89832 88977 88829 88245 84838 83331 81945 80838 80675 80203 78629 78328 77933 76507 76106 76085 74735 72691 70053 69808 68743 68632 68037 65938 65934 65586 64110 63657 61819 61541 60803 60241 59027 58303 57796 56218 55076 53437 53173 52425 51812 49121 48405 48381 48279 45446 45147 44950 44171 42308 40561 39322 39147 36988 36760 35167 33350 33113 32408 32263 32165 29438 23058 21508 21271 20177 18997 18304 16594 15458 15169 15168 14632 13439 12743 12389 10843 10522 9897 9548 9504 7824 7548 6222 4891 2635 601 3048 3804 5505 9638 9799 9801 11022 11758 11908 12198 14367 14720 15526 17302 17428 19500 22804 23604 23721 24370 24639 25335 27050 28582 29815 31873 33612 35434 36740 37612 37846 37894 40373 41706 41887 41988 42247 42762 43746 45076 46952 47322 47332 52377 52437 53321 53698 53979 54122 54528 55687 55913 56183 56902 58622 59152 60166 60881 61177 62028 63546 64013 64235 64641 64959 65116 65616 69518 70075 70453 72046 72318 73194 74467 75189 75390 80549 80853 81731 82460 82766 82783 83935 84158 85437 87731 88289 89115 92384 93009 93476 94678 95827 96467 97422 98262 101989 105307 105865 108804 109594 109969 110155 111809 112681 117822 119397 120148 120409 121515 122761 123755 123847 124053 124116 126142 126643 128411 129737 130074 130623 130804 134247 135266 135340 136283 138365 138729 139661 139736 139840 141305 144703 147133 147154 148121 148152 148536 148771 149061 152026 153597 153616 154322 155146 155389 156221 164710 165300 165709 165981 168566 171065 171573 173777 175642 177049 177922 177980 179100 180748 182019 182089 182698 184052 184206 186003 186203 188871 192385 194615 196324 196338 198910 199112 199691 200071 200794 201241 202821 203236 204040 205660 205759 207492 207619 207975 208434 209064 209684 210308 210520 211121 211947 212143 215145 216832 218824 218855 220100 220818 224540 228282 230070 230952 231381 232948 233526 234546 234696 237125 237370 238100 239974 240472 240573 241028 244122 245144 245994 246120 246206 246390 248469 249290 251152 251423 254180 255510 257006 258569 259513 259934 261634 264535 265408 266258 268474 268616 269625 270195 270530 271406 272807 273503 274288 274817 275608 275685 276497 276993 277429 278211 278244 279191 279428 281976 283273 285372 285744 287291 287302 291542 292286 293960 294746 294811 295574 296241 296418 296437 297877 298506 298506 299696 300144 300603 301205 301465 302712 303234 304010 304873 308497 310092 310955 311349 311816 311942 311980 312370 314039 314059 318369 319038 321539 321725 323121 324525 324581 325908 327902 328120 328651 329548 333008 333131 333801 334412 337063 337868 338427 340147 340226 340727 341707 342235 345195 345307 345783 345847 346694 347654 348464 349423 349652 349750 349901 350577 350848 351542 352549 353674 354792 356376 356935 357869 359419 359862 360008 361056 362357 362658 365739 365746 367055 369224 369377 371430 371807 372828 373721 374072 375521 376301 377225 378676 379384 380040 380269 381140 381577 381720 382420 383366 384071 384839 385634 386715 387057 388587 388924 391019 391591 395263 396989 397827 398012 399431 400390 402127 402137 403437 403833 404356 404596 405508 405627 407261 407771 409684 411187 411317 411352 411837 415821 417289 417421 418494 418581 419327 420667 420816 421081 423286 423767 424197 425830 426445 427080 427176 428386 429146 429389 429952 430175 430306 432539 433050 433469 433602 433873 434922 435066 435680 435819 436079 436608 436669 437680 438817 439027 440002 441910 442672 444100 444415 446243 448873 451867 455114 455552 455556 457662 458851 460665 461043 465177 465782 466517 466893 467998 470607 471472 471625 471886 472176 474003 474531 474937 475354 476728 477631 477940 478315 478394 478423 478779 479986 480021 480380 480635 481350 481611 481911 482362 483173 485010 486655 486979 487648 493944 494672 498660 499575 500586 501127 501244 502689 503584 503595 504645 505935 506049 506092 506355 508093 509031 512501 512768 513416 513443 514572 515466 517991 518798 519821 520363 521592 522374 522619 524216 526095 526374 526479 527322 527492 528400 529812 531115 531366 532854 532863 535532 535566 536947 538242 539320 539605 540246 540600 542938 543424 543936 545210 547758 550330 550771 554073 555619 555965 556401 556639 557049 557477 558794 558862 559042 559076 559346 560418 560792 560803 564091 564830 565520 566761 567370 570361 578046 578679 578806 579290 580579 581029 581508 583274 583935 585874 586227 586963 588915 589626 589653 590711 590945 591131 591352 592063 593526 595108 595603 595747 596339 598371 599951 600203 601657 603405 603733 605430 605708 606509 606590 606703 609697 613891 614610 615024 615245 615651 616278 616514 617221 618246 618672 619359 620518 621765 622241 624703 625723 630818 630933 631821 632037 634993 636359 637222 638689 639148 639688 640058 640248 640265 640646 641052 642609 642616 642775 644272 645175 646865 647125 648008 648921 650131 650537 650782 651281 651512 651662 653784 654345 655818 657088 657519 657557 657794 657800 659485 659576 659647 660690 660835 660932 662066 664508 664539 666074 666207 666895 667297 668417 669248 670046 670748 672030 672894 673979 674347 675591 675835 676775 677107 682511 683614 687173 687964 688640 690645 691454 691906 692300 692565 693342 693685 693936 697899 698334 700682 701589 701850 701943 702306 702977 703155 704117 709678 712286 712383 712480 713193 714010 715077 715441 715624 716282 716324 716361 716712 717428 717973 718104 718769 720100 721115 721308 725938 726618 726716 727469 728503 732026 732544 734233 734564 735576 738129 738777 739205 740342 741100 743800 744809 746569 747024 747508 748631 749277 751877 752338 752693 753288 753365 753921 754395 758643 760895 762014 762146 762959 763955 766752 766865 766909 767142 768991 770863 772414 772963 773883 774614 775519 776232 776655 777736 778178 779224 779927 780246 780767 781300 783610 783633 784110 785807 787925 789218 792709 792788 794035 794406 795406 795427 796259 800379 800657 800697 801022 801897 803660 804434 804582 806309 806917 807943 809587 809981 810141 810155 811546 811843 816104 817429 818450 818658 819793 820056 823555 825444 825854 825929 829216 829621 830428 830957 831858 831868 832537 833576 834926 835652 836153 836207 837197 837258 837759 837870 838525 839566 839570 840464 843138 843390 844714 844873 845495 847352 849185 849834 850084 850613 850760 852760 855557 855656 856713 857427 860386 864200 867433 868105 870714 870968 871531 872178 873828 874151 875178 875718 878013 878020 878129 878148 878242 878495 879564 879798 879927 880185 880272 880610 882085 882461 884667 889212 889544 891594 892742 893301 893354 893388 894000 894684 894809 895084 896062 897034 897713 898176 900951 900974 901554 902236 902957 903393 904431 905655 906126 907600 910015 910367 915264 917752 918721 919505 919742 924531 925134 928769 930670 931513 932822 933469 935409 935717 936153 937125 938749 938854 941705 944271 946548 949300 951882 952202 953193 954447 956453 956628 960860 965020 967110 967680 968235 969758 969779 969951 971256 971535 972038 976473 978182 978569 979507 982565 983122 983547 984560 984787 985547 985986 986093 986227 987198 988650 988662 991933 992353 993339 993623 994368 995294 995871 996131 996958 996965 997045 998117 998654 998826 998854
