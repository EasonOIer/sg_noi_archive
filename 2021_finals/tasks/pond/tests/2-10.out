1133389702915